server/api/crud/model_monitoring/deployment.py

```diff
@@ -586,14 +599,23 @@ def is_monitoring_stream_has_the_new_stream_trigger(self) -> bool:
             return False
         return True
 
-    def _create_tsdb_application_tables(self) -> None:
-        """
-        Each project writer service writes the application results into a single TSDB
-        table and therefore the target table is created during the writer deployment
-        """
-        mlrun.model_monitoring.get_tsdb_connector(
-            project=self.project,
-        ).create_tsdb_application_tables()
+    @staticmethod
+    def _create_tsdb_tables(project: str):
```

Comment:
Passing the project is redundant. It's a part of the object.
See the previous version, where this method is not static

Reply:
fhttps://github.com/mlrun/mlrun/pull/5592/commits/f0ea2d602c946f0cd9d765666427072cf4c0aa02